Competition/src/main/cpp/Lift.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Lift.h"
+
+Lift::Lift() : liftMtrLeft{LiftConstants::VICTOR_ID_LEFT_Lift}, 
+             liftMtrRight{LiftConstants::VICTOR_ID_RIGHT_Lift},
+             liftServoLeft{LiftConstants::SERVO_ID_LEFT_Lift},
+             liftServoRight{LiftConstants::SERVO_ID_RIGHT_Lift},
+             operatorController(NULL) {
+    initLift();
+}
+
+void Lift::setController(frc::XboxController* controller) {
+    operatorController = controller;
+}
+
+void Lift::initLift() {
+    
+    liftMtrLeft.SetInverted(false);
+    liftMtrRight.SetInverted(false);
+}
+
+void Lift::setDefaultState() {
+    state.liftState = LiftState::DISABLED;
+
+    resetState();
+}
+
+void Lift::assessInputs() {
+    // Prevent controller segfault
+    if (!operatorController) {
+        return;
+    }
+
+    if (operatorController->GetY(frc::GenericHID::kRightHand) > 0.05) {
+        state.liftState = LiftState::EXTEND;
+        state.rightJoystickY = operatorController->GetY(frc::GenericHID::kRightHand);
+
+        state.leftServoCurrent = liftServoLeft.Get();
+        state.rightServoCurrent = liftServoRight.Get();
+    } 
+    else if (operatorController->GetY(frc::GenericHID::kRightHand) < -0.05) {
+        state.liftState = LiftState::RETRACT;
+        state.rightJoystickY = operatorController->GetY(frc::GenericHID::kRightHand);
+
+        state.leftServoCurrent = liftServoLeft.Get();
+        state.rightServoCurrent = liftServoRight.Get();
+    }
+    else {
+        state.liftState = LiftState::DISABLED;
```

Comment:
Should set the right joystick Y values to 0 in the else case - otherwise your joystick values will always sit outside the deadband